migration/20160913-3-remove-unresolved-3m-identifiers-from-queue.sql

```diff
@@ -0,0 +1,5 @@
+delete from coveragerecords
+    where operation = 'resolve-identifier' and
+    identifier_id in (
+        select id from identifiers where type in ('3M ID', 'Bibliotheca ID')
+    );
```

Comment:
There are 17,875 UnresolvedIdentifiers with exceptions connected to 3M IDs. In the migration created on 20160830, they'll be tranformed into CoverageRecords that will never be resolved because the IdentifierResolutionCoverageProvider does not cover 3M IDs.